packages/orangutan/webpack.config.js

```diff
@@ -88,14 +88,19 @@ BundleListPlugin.prototype.apply = function(compiler) {
       const targetPackage = packagesList.find(function(element) {
         return filename.indexOf(`${element}`) !== -1
       })
+
+      const scriptSrc = isProduction
+        ? `${cdnLinkPrefix}/${filename}`
+        : `/dist/${filename}`
```

Comment:
`/dist/`  -> `distDir` ?

Reply:
`distDir = './dist'` which doesn't equal to `/dist`